exporters/prometheus/exporter.go

```diff
@@ -355,9 +355,15 @@ func createInfoMetric(name, description string, res *resource.Resource) (prometh
 }
 
 func createScopeInfoMetric(scope instrumentation.Scope) (prometheus.Metric, error) {
-	keys := scopeInfoKeys[:]
+	attrs := []attribute.KeyValue{
+		attribute.String("otel_scope_name", scope.Name),
+		attribute.String("otel_scope_version", scope.Version),
+	}
+	attrs = append(attrs, scope.Attributes.ToSlice()...)
+
+	keys, values := getAttrs(attribute.NewSet(attrs...), [2]string{}, [2]string{}, keyVals{})
```

Comment:
I thought about refactoring `getAttrs` into two functions like:

```go 
func getAttrs(attrs attribute.Set) ([]string, []string)

func getAttrsWithAppend(attrs attribute.Set, ks, vs [2]string, resourceKV keyVals) ([]string, []string)
```

but I resisted as I thought it may be too much in a single PR and also we currently have only `getAttrs(XYZ), [2]string{}, [2]string{}, keyVals{})` call in two places (`createInfoMetric`, `createScopeInfoMetric`).

Let me know if I should address it in this PR, separate PR, or should I leave as it is.

Reply:
@dashpole, I am asking for your guidance and preferences 🙏

Reply:
Here is my proposal on how to refactor it: https://github.com/open-telemetry/opentelemetry-go/pull/5937